src/System.Console/tests/ReadAndWrite.cs

```diff
@@ -30,6 +30,14 @@ public static void WriteOverloads()
             Console.SetOut(savedStandardOutput);
         }
     }
+    
+    [Fact]
+    [OuterLoop]
+    public static void WriteToOutputStream()
+    {
+        Stream outStream = Console.OpenStandardOutput();
+        outStream.Write(new byte[] { }, 0, 0);
```

Comment:
Yeah, it doesn't need to be outerloop.

This test is so that we can test this line:
```
1182: if (bytes.Length == 0)
1183:    return Interop.mincore.Errors.ERROR_SUCCESS;
```
in System.Console.ConsolePal.Windows. My understanding is that it does not matter which stream is used since we just want to get at the underlying WindowsConsoleStream.